scripts/select_appointments.sql

```diff
@@ -1 +1,8 @@
 -- retrieve appointment information
+SELECT COUNT(*), a.patient_given_name, a.patient_surname
+FROM appointments a
+INNER JOIN doctors d
+  ON p.doctor_id = d.id
+WHERE d.specialty <> 'General practice'
+GROUP BY d.specialty
+ORDER BY COUNT(*) ASC;
```

Comment:
Hmm..what is the question asking exactly? "Write a query to retrieve the names and appointment times for everyone seeing Dr. Hubbard." Is that what `COUNT` gives us? 

Here's 1 possible solution:
```sql
SELECT p.surname, a.date, a.time
  FROM patients p
  INNER JOIN appointments a ON a.patient_id = p.id
  INNER JOIN doctors d ON a.doctor_id = d.id
    WHERE d.surname = 'Hubbard';

SELECT d.surname, a.date, a.time
  FROM doctors d
  INNER JOIN appointments a ON a.doctor_id = d.id
  INNER JOIN patients p ON a.patient_id = p.id
    WHERE p.surname = 'Wilcox';
```